tareas/Andres_Sealtiel/T3_And_Seal.jl

```diff
@@ -0,0 +1,297 @@
+# -*- coding: utf-8 -*-
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .jl
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.11.4
+#   kernelspec:
+#     display_name: Julia 1.6.2
+#     language: julia
+#     name: julia-1.6
+# ---
+
+# # Temas Selectos de Física Computacional
+# ## Tarea 3
+# ### Andres Ramos A. - Sealtiel Pichardo J.
+# ### Ejr. 1
+
+using Plots
+using LaTeXStrings
+
+"""
+    F(x)
+Función que se define en los puntos x = [1, 5] tal que F(1)= 3, F(2) = 5, F(3) = 4, F(4) = 2,
+F(5) = 1 y en lo demás F es lineal.
+"""
+function F(x)
+    @assert 1 ≤ x ≤ 5   ### assert comprueba la condición a la izq
+    if x ≤ 2
+        return 2x + 1
+    elseif x ≤ 3
+        return -x + 7
+    elseif x ≤ 4
+        return -2x + 10
+    elseif x ≤ 5
+        return -x + 6
+    end 
+end
+
+"""
+    Fⁿ(x, n)
+Función que calcula el ``n``-ésimo iterado de la función F(x) y lo evalua en ``x``.
+"""
+function Fⁿ(x, n)
+    for i in 1:n
+        x = F(x)
+    end
+    return x
+end
+
+xs = 1:0.01:5
```

Comment:
Entendieron mal la función, la función solo toma números enteros, les sugiero que solo cambien esta parte:
```suggestion
xs = 1:1:5
```
hcaen eso y el ejercicio 1 queda aceptado.